ocs_ci/ocs/cluster.py

```diff
@@ -20,6 +20,9 @@
 logger = logging.getLogger(__name__)
 
 
+DEFAULT_TIMEOUT = 1200
```

Comment:
Hi Raz,
Some how I am stumbling upon this trivial timeout part.
This value is not used as is instead it will be scaled based on number of pods in the cluster, like this inside cluster_health_check() function.
```python
       if timeout == DEFAULT_TIMEOUT:
            # Scale timeout only if user hasn't passed any value
            timeout = 10 * len(self.pods)
```
But definitely its misleading. So If I want the scaled value to be default , I can't pass it in cluster_health_check(timeout=?) as default value (because I can't put expression for default param value) at the same time I don't want this value to be mandatorily passed by user.

So here is what I want
```
if user_passed_timeout:
    timeout=user_value
else 
    timeout=(no_of_pods * 10)
```
I can't think of anything apart from using **kwargs, Can you suggest some idea to handle this elegantly ?

Reply:
@shylesh you can pretty much do exactly what you do in your code example. Have a timeout parameter that defaults to None, users can then change this value to set their own timeout. Then if the timeout isn't passed, you calculate it based on the number of pods. 

Something like this:
```python
def calc_timeout(no_of_pods=4, timeout=None):
    return timeout or no_of_pods * 10
```

Reply:
Done